### ISSUE TITLE
Group commands together (entities & folders)

### ISSUE THREAD

**Issue by jamesread** [OliveTin/OliveTin#16]
Originally requested by samcro1967 on Discord. 

samcro1967: could you add drop downs as an option.  An example use case would be service management.  In config.yaml we would configure an action for systemctl specify dropdown  1 with values of stop, start, restart, and in drop down 2 specify smbd and rsyslog as an example.  Then in the GUI you can select the service from drop down 1 and the action from drop down 2.  Docker commands would be another good example.  That way instead of having multiple buttons on the screen for each service and action they could be consolidated into fewer.

@jamesread : Yep, this is something I started work on in the early days of OliveTin, but never finished off. I had this idea of "Entities" - things that you could start, stop, or basically just group a bunch of actions into - like containers, services, etc. 

There is some code in OliveTin for handling this still today, but it won't work just yet - it's something I need to work on. 

The config syntax was going to look like this; 

```
entities:
  - title: Minecraft Container
    icon: docker
    actions:
      - title: Start
        icon: start
        shell: sleep 100

      - title: Stop
        icon: stop
        shell: sleep 100 
```

samcro1967: I might collapse it just to make the config.yaml shorter to something like below. 
```
entities:
  - title: Docker
    icon: docker
    actions:
      - command:  docker
      - subcommand1: start, stop, restart
      - subcommand2: plex, wiki.js
 - title: System Services
    icon: gear
    actions:
      - command: systemctl
      - subcommand1: start, stop, restart
      - subcommand2: smbd, rsyslogd
```

**Comment by qazero** [comment]
Any update regarding this? That would be awesome :)

**Comment by jamesread** [comment]
Hey there, I'd still like to do this, but unfortunately just have not had the time. I'm due to take some days off work at the end of February and I hope to use that time to specifically work on OliveTin. :-)

**Comment by qazero** [comment]
Looking forward to it. Thanks for all your work on this amazing project!

**Comment by jamesread** [comment]
Hey, I've just had some time to work on this, and wanted to share a development video that calls out some relevant changes. Does this fix the issue for you?

https://youtu.be/vWAD04Wpa8U

**Comment by jamesread** [comment]
@qazero , so this is now released, in 2024.02.27. 

Dropdowns: https://docs.olivetin.app/args-dropdowns.html 

Creating a Container Control Panel: https://docs.olivetin.app/container-control-panel.html
